scarb/src/compiler/compilation_unit.rs

```diff
@@ -91,6 +91,10 @@ impl CompilationUnit {
         ws.target_dir().child(self.profile.as_str())
     }
 
+    pub fn is_cairo_plugin(&self) -> bool {
```

Comment:
Have you considered using an `enum` for representing compilation units? I'm not saying you have to use them, just wondering.

Reply:
Done in #1155